### PR TITLE
Layers [bt4LE, dot15d4, sixlowpan] import & refactor

### PR DESCRIPTION
**Pull request by gpotter2**
This PR currently:
- imports 802.14.5 from scapy-com
- imports SixLoWPAN from scapy-com
- imports bluetooth 4 LTE from scapy-com
- fixes SixLoWPAN, Dot15D4 for Python 3, adds ZEP link protocols, fixes existing fields, reformat code, add docstrings
- finish a few TODOs (some remaining)
- fixes already existing tests + add tests + PY3

overwrites https://github.com/secdev/scapy/pull/1179

### CODE REVIEW

File: scapy/layers/bluetooth4LE.py [hunk 1]
```diff
@@ -0,0 +1,251 @@
+## This file is for use with Scapy
+## See http://www.secdev.org/projects/scapy for more information
+## Copyright (C) Airbus DS CyberSecurity
```
Comment (gpotter2) [inline scapy/layers/bluetooth4LE.py]: Sounds like a quite reliable source

**Comment by gpotter2** [comment]
Still in WIP: need to add Bluetooth LE tests

I also have no idea why tests fail on non Sudo builds.

Finally, this code should be the last things that are not already in scapy but in avait-com

**Comment by gpotter2** [comment]
(Not for 2.4.0)

**Comment by guedou** [comment]
@gpotter2 can I review this PR?

**Comment by gpotter2** [comment]
(I have a few things to finish).

Update:
- PR contains https://bitbucket.org/secdev/scapy-com/pull-requests/3/long-address-display-format-change-from-6/diff
- PR contains https://bitbucket.org/secdev/scapy-com/pull-requests/6

**Comment by gpotter2** [comment]
@guedou Ready to be reviewed

**Comment by gpotter2** [comment]
@guedou PR ready to be reviewed!

(flake8 fixed + tests pass on my Travis account)

**Pull request merged**